stockroom/utils.py

```diff
@@ -2,6 +2,39 @@
 import types
 from pathlib import Path
 
+from rich import box
+from rich.console import Console
+from rich.table import Table
+
+# init console object to be used throught.
+console = Console()
```

Comment:
Can't we create `Console` object when it is necessary instead of creating here once and importing that everywhere?

Reply:
this was the recommended approach. I don't know the exact reason but I guess since the console object has attributes like width and height this is needed for maintaining the same everywhere.